diff --git a/html.go b/html.go
index 284c871..ab586fd 100644
--- a/html.go
+++ b/html.go
@@ -786,7 +786,7 @@ func (r *HTMLRenderer) RenderNode(w io.Writer, node *Node, entering bool) WalkSt
 		if entering {
 			align := cellAlignment(node.Align)
 			if align != "" {
-				attrs = append(attrs, fmt.Sprintf(`align="%s"`, align))
+				attrs = append(attrs, fmt.Sprintf(`style="text-align: %s;"`, align))
 			}
 			if node.Prev == nil {
 				r.cr(w)
